src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public ColorDiff getRandomColor() {
+        ColorDiff[] clr = ColorDiff.values();
+        Random random = new Random();
+        int n = random.nextInt(7);
```

Comment:
Make a random variable class level as in the photo. Also create constant for 7
<img width="988" alt="Screen Shot 2022-12-17 at 6 30 38 PM" src="https://user-images.githubusercontent.com/90692308/208251813-f0d2f400-41fc-4440-ae06-a9f57431e280.png">